pkg/mixer/lint.go

```diff
@@ -33,88 +33,89 @@ type LintOptions struct {
 }
 
 func Lint(w io.Writer, filename string, options LintOptions) error {
-	vm := NewVM(options.JPaths)
```

Comment:
Is there a reason we don't "reuse" the same VM?

Reply:
Reusing would actually be quite beneficial, because both evaluations would share cache then. This means that fields the first evaluation already evaluates don't need to be re-evaluated for the second run

Reply:
Probably out of scope of this PR however

Reply:
I removed the reuse as they now run in parallel, and I was under the impression the VMs are not safe for concurrent use.

Reply:
I've tried hard in the past to make it break, and I didn't. Ran ~20 evals on the same vm in parallel, seemed to work

Reply:
Alright. If it becomes a bottleneck in the future we know about an easy fix then. Let's go with it for now.